### PR TITLE
Rename datasafe simple adapter spring test files

### LINKED ISSUE #296: Rename datasafe-simple-adapter-spring Test files
Renaming datasafe-simple-adapter-spring Test files within datasafe-simple-adpater which perform the integration test to the IT naming convention to easily identify integration tests.
subtask #289

### PR DESCRIPTION
**Pull request by AssahBismarkabah**
Renamed Datasafe simple Adapter spring test files to separate integration tests from unit tests using the profile ID `IT`

### CODE REVIEW

**Review by Thendo20 — APPROVED** [review]
@AssahBismarkabah It looks good to me

File: datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleAdapterFeatureIT.java [hunk 1]
```diff
@@ -38,8 +38,8 @@
 import static org.junit.Assert.assertTrue;
 
 @Slf4j
-class SimpleAdapterFeatureTest extends BaseMockitoTest {
-    
+class SimpleAdapterFeatureIT extends BaseMockitoTest {
```
Comment (max402) [inline datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleAdapterFeatureIT.java]: This class doesn't use docker and works quite fast.

File: datasafe-simple-adapter/datasafe-simple-adapter-spring/src/test/java/de/adorsys/datasafe/simple/adapter/spring/InjectionForFilesystemIT.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import org.springframework.test.context.ActiveProfiles;
 
 @ActiveProfiles("filesystem")
-public class InjectionForFilesystemTest extends InjectionIT {
+public class InjectionForFilesystemIT extends InjectionIT {
```
Comment (max402) [inline datasafe-simple-adapter/datasafe-simple-adapter-spring/src/test/java/de/adorsys/datasafe/simple/adapter/spring/InjectionForFilesystemIT.java]: it's a simple test

**Review by max402 — CHANGES_REQUESTED** [review]
SimpleDatasafeAdapterTest this test uses minio docker image and quite heavy. So should be IT

**Comment by AssahBismarkabah** [comment]
updated @max402

File: datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleAdapterFeatureTest.java [hunk 3]
```diff
@@ -39,7 +39,7 @@
 
 @Slf4j
 class SimpleAdapterFeatureTest extends BaseMockitoTest {
-    
+
```
Comment (max402) [inline datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleAdapterFeatureTest.java:42]: make sure that classes remain untouched when undoing changes

**Review by max402 — APPROVED** [review]

**Pull request merged**